### PR TITLE
修正例外錯誤的回傳

### PR DESCRIPTION
**Pull request by markhsumark**

### CODE REVIEW

**Comment by markhsumark** [comment]
加入weatherhandler
例外錯誤擷取

File: src/main/java/handler/WeatherHandler.java [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+            StringBuilder sb = new StringBuilder();
+            while (line != null) {
+                sb.append(line).append(",");
+                line = buf.readLine();
+            }
+            buf.close();
+            data= sb.toString();
+            return data.split(",");
+        }catch(IOException e){
+            System.out.println(e+" =>The File may not exist");
```
Comment (asas1asas200) [inline src/main/java/handler/WeatherHandler.java:90]: Don't print any information in handler, you should use return String instead it.

Comment (asas1asas200) [reply src/main/java/handler/WeatherHandler.java:90]: Use ```System.out.println()``` may influence the output layout such like if we want to run all handlers in parallel.

**Comment by markhsumark** [comment]
例外處理修正
原本的getCurrentWeather ->produceCurrentWeather
add getCurrentWeather :(return current_weather)
tostring 改變獲取資料方式

**Comment by asas1asas200** [comment]
LGTM

**Comment by asas1asas200** [comment]
But next time I hope you pull request to `mark-feat-weather` branch.

**Pull request merged**